### ISSUE TITLE
Crashes only in REPL

### ISSUE THREAD

**Issue by floriancargoet** [FrontierPsychiatrist/node-spotify#77]
`node-spotify` works fine when I execute it in a file: `node app.js` but always crashes when used from the REPL.
```
var nodeSpotify = require('node-spotify');
var config = { 
  appkeyFile: "spotify_appkey.key"
};
var spotify = nodeSpotify(config); // works in script, segfaults in REPL
```

Reproduced on Ubuntu 64bits, OS X and Raspbian.
How can I help?

**Comment by floriancargoet** [comment]
Another test:
```
var s = require('node-spotify');
var spot = s({appkeyFile:'spotify_appkey.key'}); // doesn't crash since I didn't forget the var keyword

spot.version // works fine
spot // crashes in REPL, works in script
```

**Comment by alice0meta** [comment]
I noticed this issue myself - it seems that attempting to print the spotify object reliably segfaults.

**Comment by floriancargoet** [comment]
You're right, that's why it crashes only in the REPL when you omit the var keyword and in both the REPL and the script when you try to print it.

**Comment by FrontierPsychiatrist** [comment]
Ah, node.js prints the return value of each expression and

```
a = 1
```

is an expression!  So this bug will be fixed with #79
